docs/src/kernel/advanced-instructions.md

```diff
@@ -2,26 +2,30 @@
 
 ## User-Mode Unit Tests
 
-Asterinas consists of many crates, some of which do not require a VM environment and can be tested with the standard `cargo test`. They are listed in the root `Makefile` and can be tested together through the following Make command.
+Asterinas consists of many crates,
+some of which do not require a VM environment and can be tested with the standard `cargo test`.
+They are listed in the root `Makefile`
+and can be tested together through the following Make command.
 
 ```bash
-make test
+/asterinas$ make test
 ```
 
 To test an individual crate, enter the directory of the crate and invoke `cargo test`.
 
 ### Kernel-Mode Unit Tests
 
-Many crates in Asterinas do require a VM environment to be tested. The unit tests for these crates are empowered by our [ktest](framework/libs/ktest) framework.
+Many crates in Asterinas do require a VM environment to be tested.
+The unit tests for these crates are empowered by OSDK.
 
 ```bash
-make run KTEST=1
+/asterinas$ make ktest
 ```
 
 It is also possible to specify a subset of tests to run.
 
 ```bash
-make run KTEST=1 KTEST_WHITELIST=failing_assertion,aster_frame::test::expect_panic KTEST_CRATES=aster-frame
+/asterinas/framework/aster-frame$ cargo osdk test failing_assertion
```

Comment:
It is not entire obvious that `failing_assertion` is the name of a test case. And the description above says that `OSDK` support running a subset of tests. So the example command would be better of running multiple tests.